Make apt module work with "apt-listchanges" and "apt-listbugs" installed

jgeerds opened the pull request:
The problem basically looks like this:

```
failed: [foobar.local] => (item=virtualenvwrapper) => {"failed": true, "item": "virtualenvwrapper"}
msg: 'apt-get install 'virtualenvwrapper' ' failed: E: Sub-process /usr/sbin/apt-listbugs apt returned an error code (10)
E: Failure running script /usr/sbin/apt-listbugs apt
```

This is because a bug for the new _virtualenvwrapper_ version has been reported on the debian BTS (I'm running debian sid). A other case could be that the new package has exciting news in the **NEWS** file and _apt-listchanges_ tries to show you these news. I think we should explicitly disable the _apt-listchanges_ and _apt-listbugs_ hooks through the corresponding environment variables.

jgeerds committed:
Make apt module work with "apt-listchanges" and "apt-listbugs" installed

mpdehaan commented:
Can you include steps to reproduce this problem before your fix?  I'm a bit unclear how you are invoking things.

Also, minor style thing -- can you please define this string in one statement rather than doing the "foo" \n "newline" thing with the parenthesis?

Thanks!

jgeerds commented:
Sure! I'm using the directory structure described in the "Best Practices" section in the documentation. My tasks/main.yaml looks like this:

```
- name: Install development packages
  apt: pkg={{ item }}
  with_items:
    - […]
    - virtualenvwrapper
    - […]
```

site.yaml:

```
---
- hosts: all
  roles:
    - development
```

Then I invoke the playbook like this:

```
sudo ansible-playbook -l $(hostname -f) -i hosts -c local site.yaml
```

(At the moment I use ansible only for my local computer configuration. This is why I limit to $(hostname -f) and use the local connection mode)

FYI: I've fixed the styling thing and squashed the commits together

jctanner commented:
@geerds I'd like to get more exposure and input on this from others that work with apt. Can you solicit for some feedback on the ansible-devel list?

jctanner commented:
@geerds I'm going to close this till a community wide discussion occurs on ansible-devel. We really need peer review on packing related modules before we change their fundamental behavior.
